ui/admin/app/components/form/worker/index.hbs

```diff
@@ -99,7 +99,10 @@
     <form.input
       @name='last seen'
       @type='datetime'
-      @value={{concat relativeDate ', ' customFormat}}
+      @value={{if
+        @model.last_status_time
+        (concat relativeDate ', ' customFormat)
+      }}
```

Comment:
This display value seems like a good candidate for a getter in the form js.

Reply:
We use a few template helpers for this, should I be calling them directly in the js file?

Reply:
It would simplify the template a bit.  But probably not necessary right now.